crates/precompile/src/bls12_381/g1_add.rs

```diff
@@ -36,8 +36,8 @@ pub(super) fn g1_add(input: &Bytes, gas_limit: u64) -> PrecompileResult {
     // NB: There is no subgroup check for the G1 addition precompile.
     //
     // So we set the subgroup checks here to `false`
-    let a_aff = &extract_g1_input(&input[..G1_INPUT_ITEM_LENGTH], false)?;
-    let b_aff = &extract_g1_input(&input[G1_INPUT_ITEM_LENGTH..], false)?;
+    let a_aff = &extract_g1_input(&input[..G1_INPUT_ITEM_LENGTH], false).unwrap();
+    let b_aff = &extract_g1_input(&input[G1_INPUT_ITEM_LENGTH..], false).unwrap();
```

Comment:
why `unwrap` here?

Reply:
if I use ```expect```, then what panic msg should I give? Or should I use something else altogether?